README.md

```diff
@@ -196,26 +196,34 @@ environment variables.
 
 ## Content preview
 
-Our Starter Templates are configured to make use of Next.js' [draft mode](https://nextjs.org/docs/pages/building-your-application/configuring/draft-mode). To make use of Contentful's Content Preview we requires a few changes to be made in the code, and in Contentful.  
+Once you have the Starter Template deployed on your hosting provider, you can update the Content preview URL in your
+space settings.
```

Comment:
very pedantic comment but why is space settings on it's own line? :D

Reply:
Aahh, also a mistake when copying over the new readme. Not sure why it had so many formatting issues.